README.md

```diff
@@ -90,6 +90,8 @@ To deploy it just run `kubectl apply -f secretdefinition-sample.yaml`
 | `vault.token-polling-period` | 15s | Polling interval to check token expiration time. |
 | `vault.renew-ttl-increment` | 600 | TTL time for renewed token. |
 | `metrics-addr` | `:8080` | The address to listen on for HTTP requests. |
+| `watch-namespaces` | `""` | Comma separated list of namespaces where secrets-manager is allowed to watch. By default all namespaces will be watched. |
```

Comment:
not sure about the use of the word `allowed` here (in the exclude flag), since it implies some form of authorization type of restriction, perhaps something like:
```suggestion
| `watch-namespaces` | `""` | Comma separated list of namespaces which secrets-manager will watch for Secret Definitions. By default all namespaces will be watched. |
```

wdyt?

Reply:
makes sense thanks!!